### PR TITLE
Remove ENABLE_MINIMAL options which are disabled on netcore already.

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

File: src/mono/mono.proj [hunk 1]
```diff
@@ -206,7 +206,7 @@
       <_MonoMinimal Condition="'$(Configuration)' == 'Release'">,assert_messages</_MonoMinimal>
     </PropertyGroup>
     <ItemGroup Condition="'$(TargetsBrowser)' == 'true'">
-      <_MonoCMakeArgs Include="-DENABLE_MINIMAL=ssa,com,jit,reflection_emit_save,portability,assembly_remapping,attach,verifier,appdomains,shadowcopy,security,sgen_marksweep_conc,sgen_split_nursery,sgen_gc_bridge,sgen_toggleref,logging,remoting,shared_perfcounters,sgen_debug_helpers,sgen_binary_protocol,soft_debug,interpreter,cleanup,mdb,gac,threads,eventpipe,qcalls$(_MonoMinimal)"/>
```
Comment (akoeplinger) [inline src/mono/mono.proj:209]: ssa/DISABLE_SSA isn't set on netcore from what I can see and you also didn't remove it from iOS/Android

Comment (vargaz) [reply src/mono/mono.proj:209]: ssa is disabled when the jit is disabled.

Comment (akoeplinger) [reply src/mono/mono.proj:209]: Ah ok.

**Review by akoeplinger — APPROVED** [review]

**Comment by vargaz** [comment]
Failures are unrelated.

**Pull request merged**